### DISCUSSION TITLE
Alphanumerical dot matrix display supported by the currently used model?

### DISCUSSION THREAD

**Discussion by darmach** [jomjol/AI-on-the-edge-device#3420] [Q&A] [answered]
I'm looking for solution to read the data off alphanumerical dot-matrix type of display, like [this](https://en.wikipedia.org/wiki/Dot-matrix_display)

Is that supported by the current model - was it taught to recognise these?

**Comment by SybexX** [comment, marked as answer]
https://jomjol.github.io/neural-network-digital-counter-readout/

**Reply by darmach** [reply]
Great, thank you!
